tests/distro/test_resourceDisk.py

```diff
@@ -99,12 +106,42 @@ def test_change_partition_type(self):
         # test when sfdisk --part-type exists
         with patch.object(shellutil, "run_get_output",
                           side_effect=[[0, '']]) as run_patch:
-            resource_handler.change_partition_type(suppress_message=True, option_str='')
+            resource_handler.change_partition_type(
+                suppress_message=True, option_str='')
 
             # assert
             assert run_patch.call_count == 1
             assert "sfdisk --part-type" in run_patch.call_args_list[0][0][0]
 
+    def test_check_existing_swap_file(self):
+        test_file = os.path.join(self.tmp_dir, 'test_swap_file')
+        file_size = 1024 * 128
+        if os.path.exists(test_file):
+            os.remove(test_file)
+
+        with open(test_file, "wb") as file:
+            file.write(bytes(file_size))
```

Comment:
We need to use `bytearray(file_size)` instead of `bytes(file_size)`. In Python 2, `bytes(file_size)` would just write the string representation of `file_size`, making the file size not `file_size`. This is why the assertions are failing for Python 2.